### PR TITLE
Feature: New miniplayer

### PR DESCRIPTION
**Pull request by pichfl**
Re #52 

The colors and the background are postponed until we finish the new data structure which will store the extracted colors.

This also improves the gestures for interacting with the player by adjusting their degrees of freedom and required distance. 

Re #68 - As I was messing with the LibraryViewController already, I also removed the Remote events to the app delegate.

### CODE REVIEW

File: BookPlayer/Credits.html [hunk 1]
```diff
@@ -45,7 +45,7 @@
             @bryanrezende for the Smart rewind<br />
             @e7mac for the speed controls and autoplay<br />
             @gpambrozio for the Volume Boost<br />
-            @pichfl for Design, App Icon and Refactoring<br />
+            @pichfl for Design, Artwork and Refactoring<br />
```
Comment (GianniCarlo) [inline BookPlayer/Credits.html]: same clarification should be on the `README.md`

**Comment by GianniCarlo** [comment]
I just got one comment, which is arguable 🤔. If I tap the play/pause button on the mini player, the playerVC is shown, I could make an argument that if I took the trouble to actually hit the button to pause it, instead of opening the player first and pause it, I wouldn't want the player to be shown

**Comment by pichfl** [comment]
I thought I had that added to the gestures, seems I stashed it at some point and forgot to commit

**Comment by GianniCarlo** [comment]
There's a problem, when I drag the mini player, once the player is presented, if I haven't lifted my finger, and if I continue to drag, the app will try to show the player over and over again. This won't have any visual effect, but it will generate warning logs:
```
BookPlayer[15756:2196696] Warning: Attempt to present <BookPlayer.PlayerViewController: 0x1043440f0>  on <BookPlayer.LibraryViewController: 0x1043239a0> which is already presenting (null)
```
I found this while I was trying out a haptic feedback when the show player event was triggered from dragging the mini player
```
if #available(iOS 10.0, *) {
                    let generator = UINotificationFeedbackGenerator()
                    generator.notificationOccurred(.success)
                }
```
I think the feedback adds a nice touch to it, test it out and let me know what you think

**Comment by pichfl** [comment]
I'll implement a similar reset mechanism to what I use for artwork. The vibration is a nice touch, I'll add it as well.

**Pull request merged**